go-controller/pkg/ovn/egressip.go

```diff
@@ -1600,6 +1600,14 @@ func (oc *Controller) setNodeEgressReachable(nodeName string, isReachable bool)
 
 func (oc *Controller) addEgressNode(nodeName string) error {
 	var errors []error
+	// Check if EgressIP node create failed and if does try adding it again
+	if node, ok := oc.addEgressIPNodeFailed.Load(nodeName); ok {
+		failedNode := node.(*kapi.Node)
+		if err := oc.addNodeForEgress(failedNode); err != nil {
```

Comment:
at first I was concerned that calling this would invoke functions like `createDefaultNoRerouteNodePolicies`. But as I looked at that, they seem to be idem-potent and create is actually createOrUpdate. Phew!